gslib/commands/acl.py

```diff
@@ -340,6 +346,50 @@ class AclCommand(Command):
       },
   )
 
+  def get_gcloud_storage_args(self):
+    sub_command = self.args.pop(0)
+    if sub_command == 'get':
+      if StorageUrlFromString(self.args[0]).IsObject():
+        command_group = 'objects'
+      else:
+        command_group = 'buckets'
+      gcloud_storage_map = GcloudStorageMap(gcloud_command=[
+          'alpha', 'storage', command_group, 'describe', self.args[0], '--format=json[acl]'
+      ],
+                                            flag_map={})
+      self.args = self.args[:-1]
+    
+    elif sub_command == 'set':
+      if len(self.args[1:])>1:
+        url_list = self.args[1:]
+        object_or_bucket_url = [StorageUrlFromString(i) for i in url_list]
```

Comment:
It doesn't matter if there are one or more arguments. This code should work for both cases.